### ISSUE TITLE
Question - SSO

### ISSUE THREAD

**Issue by rjamesnw** [formtools/core#558]
I like the system, but I have a client that needs an SO feature (Google specifically). Does Form Tools support 3rd-party logins (single sign-ins)?

**Comment by benkeen** [comment]
Agreed, this is becoming more and more vital. But I'm afraid it's not on the roadmap yet - the next 3 versions are pretty clearly mapped out, and they're all important functionality/features that are needed.
